src/components/Gallery/Gallery.tsx

```diff
@@ -0,0 +1,119 @@
+import React from "react";
+import { Image } from "../Image/Image";
+import { Swiper, SwiperSlide } from "swiper/react";
+import { Pagination } from "swiper/modules";
+import "swiper/css/pagination";
+import "swiper/css";
+
+import eventImage1 from "../../assets/event_diningTable.png";
+import eventImage2 from "../../assets/event_walledTent.png";
+import eventImage3 from "../../assets/event_conference.png";
+import eventImage4 from "../../assets/event_beachPopupTents.png";
+import eventImage5 from "../../assets/event_colorLights.png";
+import eventImage6 from "../../assets/event_treeLights.png";
+import eventImage7 from "../../assets/event_marathonFinishLine.png";
+import eventImage8 from "../../assets/event_groupOutdoor.png";
+import eventImage9 from "../../assets/event_SantaPhotos.png";
+
+export const Gallery: React.FC = () => {
```

Comment:
Ideally the Gallery component would accept an array of images, instead of having them defined here.

Reply:
Totally makes sense! I made that change in the new branch